src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -126,6 +118,64 @@ def make_parameter_file(
         json.dump(output, f)
 
 
+def translate_outputs(product_dir: Path, product_name: str):
+    """Translate ISCE outputs to a standard GTiff format with a UTM projection
+
+    Args:
+        product_dir: Path to the ISCE merge directory
+        product_name: Name of the product
+    """
+    ISCE2_DATASET = namedtuple('ISCE2_DATASET', ['name', 'suffix', 'band'])
+    datasets = [
+        ISCE2_DATASET('filt_topophase.unw.geo', 'unw_phase', 2),
+        ISCE2_DATASET('phsig.cor.geo', 'corr', 1),
+        ISCE2_DATASET('z.rdr.full.geo', 'dem', 1),
+        ISCE2_DATASET('filt_topophase.unw.conncomp.geo', 'conncomp', 1),
+        ISCE2_DATASET('filt_topophase.flat.geo', 'wrapped_phase', 1),
+    ]
+
+    for dataset in datasets:
+        out_file = str(Path(product_name) / f'{product_name}_{dataset.suffix}.tif')
+        in_file = str(product_dir / dataset.name)
+
+        if dataset.suffix == 'wrapped_phase':
+            cmd = (
+                f'gdal_calc.py --outfile {out_file} -A {in_file} '
+                '--calc angle(A) --type Float32 --format GTiff --NoDataValue=0 '
+                '--creation-option TILED=YES --creation-option COMPRESS=LZW --creation-option NUM_THREADS=ALL_CPUS'
+            )
+
+            subprocess.call(cmd.split(' '))
+        else:
+            gdal.Translate(
+                destName=out_file,
+                srcDS=in_file,
+                bandList=[dataset.band],
+                format='GTiff',
+                noData=0,
+                creationOptions=['TILED=YES', 'COMPRESS=LZW', 'NUM_THREADS=ALL_CPUS'],
+            )
+
+    ds = gdal.Open(str(product_dir / 'filt_topophase.unw.geo'))
+    geotransform = ds.GetGeoTransform()
+    ds = None
```

Comment:
Would this accomplish the same thing? If so, then I think it would make the intention more explicit.
```suggestion
    del ds
```

Reply:
Or just:

```python3
geotransform = gdal.Open(str(product_dir / 'filt_topophase.unw.geo')).GetGeoTransform()
```